core/new-gui/.gitignore

```diff
@@ -44,4 +44,4 @@ testem.log
 .DS_Store
 Thumbs.db
 
-environments/version.ts
+src/environments/version.ts
```

Comment:
I do see version.ts being tracked. do you want to remove it from git and also change the .gitignore to `**/environment/version.ts`?

Reply:
that change is also in this PR. The version.ts file is being deleted below.

Reply:
Github shows that you are on at outdated version of this PR, hence you can't see the newest changes. The version.ts is being deleted from git in this PR